package.json

```diff
@@ -18,21 +24,32 @@
       "last 1 safari version"
     ]
   },
+  "peerDependencies": {
+    "@sentry/browser": "^5.15.4",
+    "@sentry/integrations": "^5.15.4",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1"
```

Comment:
pin versions

Reply:
I haven't pinned these as its peer dependencies. If pinned the consumer library might end up installing multiple versions of these libraries. Does that make sense ?